test-framework/provisioning/roles/barnacle/tasks/main.yml

```diff
@@ -35,6 +35,16 @@
       dest: /etc/resolv.conf
       mode: 0644
 
+  - name: Prevent dhclient-script from overwriting resolv.conf
+    copy:
+      dest: /etc/dhcp/dhclient-enter-hooks
+      content: |
+        make_resolv_conf() {
+          exit 0
+        }
+      mode: 0744
+    when: ansible_distribution == "CentOS"
```

Comment:
This isn't needed for other distributions?

Reply:
Nope, SLES dhclient-script doesn't overwrite /etc/resolv.conf